salesagility/SuiteCRM
# Fixed #2176 - Days Dim - requesting context for translation

## Description



Removes hard-coded french translations for map markers. See original issue for screenshots.

## Motivation and Context

Issue reference: #2176 

## How To Test This

1. Enter map markers.
2. In the edit-view of a map see that "days_dim" and other hard-coded translations have been removed.

## Types of changes


### Final checklist
